Parse/src/main/java/com/parse/ParseRESTCommand.java

```diff
@@ -249,6 +249,10 @@ protected ParseHttpBody newBody(ProgressCallback uploadProgressCallback) {
     }
   }
 
+  public Task<JSONObject> executeAsync() {
+    return executeAsync(ParsePlugins.get().restClient());
+  }
+
```

Comment:
This defeats the purpose of injection as it makes it difficult to unit test execution paths that use this method.

In reality, we should be passing the client through to anything that was using `ParseRequest#executeAsync()` and `ParseRequest#executeAsync(ProgressCallback, ProgressCallback, CancellationToken)`.